addons/base_automation/models/base_automation.py

```diff
@@ -61,6 +61,9 @@ class BaseAutomation(models.Model):
     filter_domain = fields.Char(string='Apply on', help="If present, this condition must be satisfied before executing the action rule.")
     last_run = fields.Datetime(readonly=True, copy=False)
     on_change_fields = fields.Char(string="On Change Fields Trigger", help="Comma-separated list of field names that triggers the onchange.")
+    trigger_field_ids = fields.Many2many('ir.model.fields', string='Watched fields',
```

Comment:
I was wondering whether we could merge `trigger_field_ids` and `on_change_fields`.
They are both used in a similar way, except for the empty value: it always triggers the action for create/write, while it never triggers it for onchange. But an empty list in the case of onchange does not make sense...